src/Components/FavoritesContainer/FavoritesContainer.scss

```diff
@@ -38,3 +39,9 @@
 .announce-title {
   @include font-family-size(Bebas Neue, 54px);
 }
+
+@media screen and (min-device-width: 0px) and (max-device-width: 800px) {
+  .fav-header {
+    flex-direction: column;
+  }
```

Comment:
Add get-more-inspo to adjust top margin.
Does the button seem off-center to y'all?  It could just be the styling that makes it seem so.